recipes/scikit-bio/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set version = "0.4.2" %}
+
+package:
+  name: scikit-bio
+  version: {{ version }}
+
+source:
+  fn: scikit-bio-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-bio/scikit-bio-{{ version }}.tar.gz
+  md5: bbbe704410c59083e9eb65fa9d01a50d
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bz2file >=0.98
+    - lockfile >=0.10.2
+    - cachecontrol >=0.11.5
+    - contextlib2 >=0.4.0
+    - decorator >=3.4.2
+    - future >=0.14.3
+    - ipython >=3.2.0
+    - matplotlib >=1.4.3
+    - natsort >=4.0.3
+    - numpy >=1.9.2
+    - pandas ==0.17.*
```

Comment:
Got it, I was a bit confused by the PEP itself, thanks for clearing this
out.

On (Jul-25-16| 9:05), jakirkham wrote:
> > +
> > +requirements:
> > +  build:
> > +    - python
> > +    - setuptools
> > +    - bz2file >=0.98
> > +    - lockfile >=0.10.2
> > +    - cachecontrol >=0.11.5
> > +    - contextlib2 >=0.4.0
> > +    - decorator >=3.4.2
> > +    - future >=0.14.3
> > +    - ipython >=3.2.0
> > +    - matplotlib >=1.4.3
> > +    - natsort >=4.0.3
> > +    - numpy >=1.9.2
> > +    - pandas ==0.17.*
> 
> Sorry that was a bit vague now that I'm re-reading it. Simply remove `==`.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/conda-forge/staged-recipes/pull/1071/files/82b53a2211c317aff8d87d903d4e580ee20e7252#r72092881